### PR TITLE
chore(lorempicsum.avatar): deprecate

### PR DESCRIPTION
**Pull request by xDivisionByZerox**
Listed as a todo in #1044.

This PR deprecates `faker.image.lorempicsum.avatar()` in favor of `faker.internet.avatar()`.

### CODE REVIEW

**Comment by xDivisionByZerox** [comment]
The failing pipeline is due to some problem in the post-install script for `simple-git-hooks`. So not due to my implementation changes.

File: src/modules/image/providers/lorempicsum.ts [hunk 1]
```diff
@@ -73,12 +74,21 @@ export class LoremPicsum {
...
    * faker.internet.avatar()
    * // 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/315.jpg'
+   *
+   * @deprecated
    */
-  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with lorempicsum.
   avatar(): string {
+    deprecated({
+      deprecated: 'faker.image.lorempicsum.avatar()',
+      proposed: 'this.faker.internet.avatar()',
```
Comment (import-brain) [inline src/modules/image/providers/lorempicsum.ts:88]: Should `this` be here?

Comment (xDivisionByZerox) [reply src/modules/image/providers/lorempicsum.ts:88]: Yeah probably copied it from the implementation below. You are right.

**Review by ST-DDT — APPROVED** [review]

**Review by import-brain — DISMISSED** [review]

File: test/image.spec.ts [hunk 2]
```diff
@@ -62,6 +62,21 @@ describe('image', () => {
             )
         ).toBeTruthy();
       });
+
+      it('should log a deprecation message', () => {
```
Comment (Shinigami92) [inline test/image.spec.ts:66]: Please revert this file, as we don't need these direct tests anymore

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: test/image.spec.ts (hunk 2 above)
Comment (xDivisionByZerox) [reply test/image.spec.ts]: done 9eeddb9

**Review by import-brain — APPROVED** [review]

**Review by Shinigami92 — DISMISSED** [review]

File: src/modules/image/providers/lorempicsum.ts [hunk 3]
```diff
@@ -73,12 +74,21 @@ export class LoremPicsum {
...
+   *
+   * @deprecated
    */
-  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with lorempicsum.
   avatar(): string {
+    deprecated({
+      deprecated: 'faker.image.lorempicsum.avatar()',
+      proposed: 'faker.internet.avatar()',
+      since: 'v7.3.0',
+      until: 'v8.0.0',
```
Comment (ST-DDT) [inline src/modules/image/providers/lorempicsum.ts:90]:
```suggestion
      since: '7.3',
      until: '8.0',
```

Changed in preparation of #1067 .

**Review by Shinigami92 — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**